lib/hutch/message_handler.rb

```diff
@@ -0,0 +1,60 @@
+require 'hutch/message'
+
+module Hutch
+  class MessageHandler
```

Comment:
This class kind of sucks because I was pulling functionality into new classes and got tired of trying to fix tests, so this is kind of a dump of old functionality from the worker class.